config/data.js

```diff
@@ -841,6 +841,7 @@ module.exports = {
     'raw_label': 'commuters',
     'decimals': 1,
     'type': 'weighted',
+    'world_val': {'y_2019},
```

Comment:
@Vanie2020 looks like this is missing a value. Were you able to run the compass locally on this branch of the data? It looks like it would fail the build process.